packages/adapter-nuxt/src/hooks/documentation-read.ts

```diff
@@ -39,12 +39,13 @@ export const documentationRead: DocumentationReadHook<PluginOptions> = async (
 					})
 				);
 
-				useHead({
+				useSeoMeta({
 					title: page.value?.data.meta_title,
-					meta: [{
-						name: "description",
-						content: page.value?.data.meta_description,
-					}],
+					ogTitle: page.value?.data.meta_title,
+					description: page.value?.data.meta_description,
+					ogDescription: page.value?.data.meta_description,
+					ogImage: computed(() => prismic.asImageSrc(page.value?.data.meta_image)),
+					twitterCard: "summary_large_image",
```

Comment:
❓We would need the same on other adapters, right?

What is the reason to start supporting this now?
Also, twitterCard value can be `“summary”, “summary_large_image”, “app”, or “player”`, how to be sure that it's good to hard set `"summary_large_image"`? Should it come from the content of the page?

Reply:
I don't know, this is the setup we recommend with Nuxt to cover most common use cases: https://nuxt.com/docs/api/composables/use-seo-meta#usage

Reply:
Updated on Next side! Svelte manages meta on the layout level